x-pack/platform/plugins/shared/ml/public/cases/constants.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Titles for the cases toast messages
+ */
+export const TITLES = {
+  ANOMALY_TIMELINE: 'Anomaly timeline',
```

Comment:
Do these need to be internationalized?

Reply:
Done in: [#7f408c6](https://github.com/elastic/kibana/pull/203813/commits/7f408c657c636217c0c2b20650d57b6dbfc54300)